How to Conduct a Successful Interview.md

```diff
@@ -0,0 +1,39 @@
+# How to Conduct a Successful Interview
+
+When writing a blog post or article, sometimes it’s useful to include an expert opinion or personal quote to boost your credibility and to improve engagement with your content. If you’ve been assigned to conduct an interview or if you think it could benefit your writing task, follow these guidelines to successfully conduct a professional interview.
+
+- **Ask, don’t expect.**
+
+Generally, it’s best to never assume someone is free. In an email or a Slack message, ask the person politely if they would be interested in answering some questions. When sending the first message, outline what you’re working on and the topic you would like to interview them about so they know what they’re agreeing to from the beginning.
+
+- **Come prepared.**
+
+Once you’ve arranged a time to meet with your interviewee, be sure to prepare for it. Outline all the questions you need answered before the interview, and even memorize some of them so you can work them into the conversation naturally.
+
+- **Ask the right questions.**
```

Comment:
Use your research to ask unique questions, if you can google it don't ask it. If you research things that you're not confident about, feel free to confirm if that info is accurate before poking holes and getting more info on it